### PR TITLE
Draft: Appium versions

### PR DESCRIPTION
**Pull request by wswebcreation**
This new page should give customers clarity about which Appium 1 and 2 drivers are being supported in the Real and Virtual Device Cloud and which ones will be deprecated.
Customers will be informed in the UI when a version will be deprecated including steps on how to upgrade/migrate

### CODE REVIEW

File: docs/mobile-apps/automated-testing/appium/appium-versions.md [hunk 1]
```diff
@@ -0,0 +1,2769 @@
...
+
+- bugfixes will not be released for Appium 1.x anymore and require you to [migrate to Appium 2](./appium-2-migration.md)
+- Appium 1 and its packaged drivers are no longer 100% compatible with Android 13/iOS 16 and higher and require Appium 2 to run stably.
+
+Sauce Labs still supports Appium 1.x in limited versions, but we recommend [migrating to Appium 2](./appium-2-migration.md) to keep your tests up-to-date and compatible with the latest platform versions.
+:::
+
+## Selecting Appium Versions
+
+You can select a specific Appium version by using the `appiumVersion` capability as part of the `"sauce:options"`. The below examples show how to select the Appium version for your test and use Android with Chrome as an example. See our [Platform Configurator](https://saucelabs.com/products/platform-configurator#/) to help you construct your capabilities for the specific platform and Real Device/Android Emulator/iOS Simulator you want to test on.
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/appium/appium-versions.md:23]: is the platform configurator default to Appium 2 already?

File: docs/mobile-apps/automated-testing/appium/appium-versions.md [hunk 2]
```diff
@@ -0,0 +1,2769 @@
...
+    "appium:platformVersion" : "14",
+    "appium:deviceName" : "Google Pixel 7 Pro",
+    "appium:automationName": "uiautomator2",
+    "sauce:options" : {
+        # Check below for the available version
+        "appiumVersion" : "latest"
+    }
+}
+```
+
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/appium/appium-versions.md:91]: do we really need all the c# and ruby examples?

File: docs/mobile-apps/automated-testing/appium/appium-versions.md [hunk 3]
```diff
@@ -0,0 +1,2769 @@
...
+</TabItem>
+</Tabs>
+
+## Release Strategy
+
+Our Appium 2 release strategy is as follows:
+
+- On the 2nd of each month, we release a 'bundled' version of Appium 2 that includes all the latest drivers available up to the 1st of that month. This 'bundled' version allows you to maintain a stable testing environment, saving you the effort required to individually update drivers. The version is named with a timestamp in the `appiumVersion` capability, such as `appium2-YYYYMMDD`, for easy tracking. These versions have an 11-month lifespan post-release, giving you plenty of time to plan your upgrade strategy.
```
[the remark this answers is not in the document]
Comment (wswebcreation) [reply docs/mobile-apps/automated-testing/appium/appium-versions.md:136]: The versions are in the other tables

File: docs/mobile-apps/automated-testing/appium/appium-versions.md (hunk 2 above)
Comment (wswebcreation) [reply docs/mobile-apps/automated-testing/appium/appium-versions.md:91]: yes, it's a standard :(

File: docs/mobile-apps/automated-testing/appium/appium-versions.md (hunk 1 above)
Comment (wswebcreation) [reply docs/mobile-apps/automated-testing/appium/appium-versions.md:23]: yes, for EMUSIM it does, for RDC it's not defaulting. It's based on internal logic

**Pull request merged**